playground/public/preview-drupal-ce-test.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Drupal Custom Element Test</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      margin: 20px;
+    }
+    .preview-target {
+      border: 2px dashed #ccc;
+      min-height: 200px;
+      padding: 20px;
+      margin: 20px 0;
+      background: #fafafa;
+    }
+  </style>
+  <!-- Single script to initialize Nuxt app with component preview enabled -->
+  <script src="/nuxt-component-preview/app-loader.js"></script>
+</head>
+<body>
+  <h1>Drupal Custom Element Fallback Test</h1>
+
```

Comment:
let's add some note that this test only works if the module is added in the playground first